pmg/static/resources/css/style.scss

```diff
@@ -1050,3 +1090,23 @@ body.email_alerts {
     }
   }
 }
+
+@media screen and (max-width:544px) {
```

Comment:
I suggest using bootstrap's .visible-xs-block (or just .visible-xs if we're a few versions behind) - http://getbootstrap.com/css/#responsive-utilities

That saves us from writing the bulk of this css.

Also .btn-block makes buttons 100% wide.